### DISCUSSION TITLE
Update repo - with PR inluded

### DISCUSSION THREAD

**Discussion by JoseConseco** [wbthomason/packer.nvim#1093] [Q&A]
Hi, is there way to update plugin with some picked PR included  using packer? 
Something like:
```
use 'xyzz'
 branch = "pr/123"
```

I think this used to work in past?

**Comment by nat-418** [comment]
You can specify a `branch`, `tag`, and `commit`. So for a particular PR I would:

1. Make sure I am pointed at the right fork.
2. Point the `commit` to the relevant hash.
